### DISCUSSION TITLE
Safety

### DISCUSSION THREAD

**Discussion by VeryBerryYogurt** [IceHacks/SurvivCheatInjector#856] [General]
Can we get the source code to be public in the github? I'd like to make sure this is safe, and won't compromise my device. Also, could I get the server invite link?

**Comment by viridianknight7** [comment]
https://discord.gg/a7abxQPuA4

**Comment by viridianknight7** [comment]
And no, the hack is 100% malware

**Comment by viridianknight7** [comment]
Never, ever use it unless you want your computer to explode. It is an extreme safety hazard and I do not recommend downloading it.

**Reply by VeryBerryYogurt** [reply]
thanks!

**Comment by IceHacks** [comment]
Upload the exe to VirusTotal. This is what you will get: https://www.virustotal.com/gui/file/0aa18eccb9245ab43d16fa880ecf910ae5695746a167e343d79ec41af9363935/details
